src/components/position-at-box-shape-border.js

```diff
@@ -59,6 +59,8 @@ AFRAME.registerComponent("position-at-box-shape-border", {
   tick() {
     if (!this.target) {
       this.targetEl = this.el.querySelector(this.data.target);
```

Comment:
Hmm, I know this wasnt just added but we really shouldn't be doing a querySelector on every tick

Reply:
`this.target` will eventually converge to non-null (assuming the author wrote a sane selector)